__tests__/app.test.js

```diff
@@ -61,6 +61,27 @@ describe("Testing endpoints in the app.js file", () => {
         );
       });
   });
+  test("GET: 200 - request to /api/articles responds with all articles sorted by date", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.articles.length).toBe(13);
+        expect(response.body.articles).toBeSortedBy("created_at", {
+          descending: true,
+        });
+        response.body.articles.forEach((article) => {
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.topic).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.votes).toBe("number");
+          expect(typeof article.article_img_url).toBe("string");
+          expect(typeof article.comment_count).toBe("string");
+        });
+      });
```

Comment:
You could add a test to check that this doesn't contain a body property. Have a look at the docs to help you out: https://jestjs.io/docs/expect#tohavepropertykeypath-value